### PR TITLE
ARSN-445 Cherry-pick change for AWS KMS backend in CloudServer

### PR DESCRIPTION
**Pull request by nicolas2bert**
Code has already been reviewed.

### CODE REVIEW

**Comment by nicolas2bert** [comment]
ping

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "engines": {
     "node": ">=16"
   },
-  "version": "7.70.4-1",
+  "version": "7.70.4-2",
```
Comment (anurag4DSB) [inline package.json:6]: Jira ticket might need the hotfix version

**Review by anurag4DSB — APPROVED** [review]
Approving to correct

**Comment by nicolas2bert** [comment]
@bert-e approve

**Review by dvasilas — APPROVED** [review]

**Pull request merged**